packages/lb-components/src/components/rect2DOperationView/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import { connect } from 'react-redux';
+
+import { a2MapStateToProps } from '@/store/annotation/map';
+import { LabelBeeContext } from '@/store/ctx';
+import { ImgUtils, Rect2DOperation, uuid } from '@labelbee/lb-annotation';
+import { IBasicRect } from '@labelbee/lb-utils';
+
+import { usePointCloudViews } from '../pointCloudView/hooks/usePointCloudViews';
+import { PointCloudContext } from '../pointCloudView/PointCloudContext';
+
+interface IRect2DOperationViewProps {
+  mappingData?: any;
+  size: {
+    width: number;
+    height: number;
+  };
+  config: any;
+  checkMode?: boolean;
+}
+
+interface IRect2DOperationViewRect extends IBasicRect {
+  boxID: string;
+  id: string;
+  attribute: any;
+  order?: number;
+}
+
+const Rect2DOperationView = (props: IRect2DOperationViewProps) => {
+  const { mappingData = {}, size, config, checkMode } = props;
+  const { url } = mappingData;
+  const { pointCloudBoxList, setPointCloudResult, selectedID, setCuboidBoxIn2DView } =
+    useContext(PointCloudContext);
+
+  const { update2DViewRect } = usePointCloudViews();
+  const ref = React.useRef(null);
+  const operation = useRef<any>(null);
+  const currentFn = useRef<any>(null);
+
+  const handleUpdateDragResult = (rect: IRect2DOperationViewRect) => {
+    const results = currentFn.current?.(rect);
+    setPointCloudResult(results);
+  };
+
+  useEffect(() => {
+    currentFn.current = update2DViewRect;
+  }, [update2DViewRect]);
+
+  const setRects = () => {
+    let allRects: IRect2DOperationViewRect[] = [];
+    pointCloudBoxList.forEach((pointCloudBox) => {
+      const { rects = [], id, attribute, trackID } = pointCloudBox;
+      const rect = rects.find((rect) => rect.imageName === mappingData.url);
```

Comment:
You can define more Info in `IMappingImg`. 

Like add `path` in `IMappingImg`.
![image](https://github.com/open-mmlab/labelbee/assets/21312795/3e9e01d4-fced-4a4f-b45b-54d9e6122bf8)

This way is update in `sensebee`
<img width="1594" alt="image" src="https://github.com/open-mmlab/labelbee/assets/21312795/13a9346e-9368-40f5-997b-aac6928d422b">